config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -356,3 +356,52 @@ periodics:
       options:
       - name: ndots
         value: "1"
+
+- name: aws-tests
+  interval: 6h
+  agent: kubernetes
+  decorate: true
+  # extra_refs:
+  # - org: jetstack
+  #   repo: cert-manager
+  #   base_ref: master
```

Comment:
What do these do? Are they needed?

Reply:
It specifies what repository this Prow job clones when it runs.

Reply:
Thanks! Would you also know where it clones the repo? For example right now when we run the golang image we are in `/go` at the start. So if we specify this, would we be inside `/go/cert-manager` or would we still be in `/go` but have the `cert-manager` folder already cloned for us?

Reply:
From a quick check it's cloned at `/home/prow/go/src/github.com/jetstack/cert-manager` and Prow sets that as `workingDir` for the test container.

Reply:
By `workingDir` do you mean that at the launch of the container we would be at `/home/prow/go/src/github.com/jetstack/cert-manager`? If yes, then if we add another repo would the first one be the `workingDir`?

Reply:
> By workingDir do you mean that at the launch of the container we would be at /home/prow/go/src/github.com/jetstack/cert-manager? 

Yep, [the workingDir in the container spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#container-v1-core).

> If yes, then if we add another repo would the first one be the workingDir

Looking at the code [here](https://github.com/kubernetes/test-infra/blob/cc3a0f99acfab2e07213e9a45937130c79bcc7fa/prow/pod-utils/decorate/podspec.go#L795-L803) looks like that would happen by default, but it seems like we can also [set `work_dir: true` for a particular ref](https://github.com/kubernetes/test-infra/blob/cc3a0f99acfab2e07213e9a45937130c79bcc7fa/prow/apis/prowjobs/v1/types.go#L907) to configure which one should become `workingDir`.
I've not tried cloning more than one myself.